src/corelibs/U2View/src/ov_phyltree/TreeViewer.cpp

```diff
@@ -209,6 +210,10 @@ void TreeViewer::createActions() {
 
     // Print Tree
     printAction = new QAction(QIcon(":/core/images/printer.png"), tr("Print Tree..."), ui);
+#ifdef Q_OS_WIN
+    printAction->setDisabled(QPrinterInfo::availablePrinterNames().isEmpty());
```

Comment:
Printers may appear after the MSA editor is open.
You should make the check after the button is clicked and show "No printers"-like message box if there are no printers.